runtime/src/accounts_db.rs

```diff
@@ -5525,6 +5559,31 @@ impl AccountsDb {
                 }
             }
         }
+
+        let mut filler_accounts = 0;
+        if self.filler_accounts_enabled() {
+            let slots_remaining = self.filler_account_slots_remaining.load(Ordering::Acquire);
+            if slots_remaining > 0 {
+                // figure out
+                let pr = self.get_prior_root(slot);
+
+                if let Some(prior_root) = pr {
+                    let filler_account_slots =
+                        std::cmp::min(slot.saturating_sub(prior_root), slots_remaining);
+                    self.filler_account_slots_remaining
+                        .fetch_sub(filler_account_slots, Ordering::Release);
+                    let filler_accounts_per_slot =
+                        self.filler_accounts_per_slot.load(Ordering::Acquire);
+                    filler_accounts = filler_account_slots * filler_accounts_per_slot;
+
+                    // keep space for filler accounts
+                    let addl_size = (filler_accounts as u64)
+                        * ((self.filler_accounts_config.size + STORE_META_OVERHEAD) as u64);
+                    total_size += addl_size;
+                }
+            }
```

Comment:
Is it possible for `do_flush_slot_cache()` to be called simultaneous by more than one thread? Like from two banks maybe? I think flush is only called in ABS, which should process banks/flush serially, right?

Reply:
Should not be parallel. Do you have a specific concern?

Reply:
Since this is a cli enabled debug feature, I'm not too concerned about this being wrong. None of this code even runs if filler accounts are not enabled. If we add a few too many or too few accounts, that isn't the end of the world.